e2e/portalicious/pages/PaymentsPage.ts

```
@@ -182,6 +182,29 @@ class PaymentsPage extends BasePage {
     await this.exportDropdown.click();
     await this.page.getByRole('menuitem', { name: option }).click();
   }
+
+  async validateGraphStatus({
+    pending,
+    successful,
+    failed,
+  }: {
+    pending: number;
+    successful: number;
+    failed: number;
+  }) {
+    const graph = await this.page.locator('canvas').getAttribute('aria-label');
+    if (graph) {
+      const graphText = graph
+        .replace('Payment status chart.', '')
+        .replace(/\s+/g, ' ')
+        .trim();
+      expect(graphText).toContain(
+        `Pending: ${pending}, Successful: ${successful}, Failed: ${failed}`,
+      );
+    } else {
+      console.log('Graph attribute is null');
```

I think this should throw an error

---

Oh yes indeed. Thanks!